src/core_atmosphere/Registry.xml

```diff
@@ -884,8 +880,6 @@
 			<var name="h_oml_initial"/>
 			<var name="hu_oml"/>
 			<var name="hv_oml"/>
-			<var name="p_fg"/>
-			<var name="o3mr_fg"/>
 			<var name="OPPpres"/>
```

Comment:
Hi Laura,
Many thanks for your updates. I have questions about the three variables in restart stream: OPPpres, OPPdata, and OPPcoeffs

1. For my test run turning off “prognostic O3”, I see the file size is about 4 times larger than before in the new forecast file because of including these three variables. I checked the values , and they are all zero. 
Is it ok to remove these three variables from the stream, considering the disk space for our experiments especially for high resolution runs?

2. If we turn on “prognostic O3” in the future: Are these variables time-invariant fields or will the value be changed from cycle to cycle?

Many thanks,
Junmei